admin/RELEASING.md

```diff
@@ -14,21 +26,23 @@ that needs to be remembered due to the Android component's distribution.
         * We typically leave these branches around for future maintenance releases.
     4. Run `ci/package_android_release.sh` in a UNIX compatible shell
     5. (Optional) `cargo publish -p rustls-platform-verifier-android --dry-run --alow-dirty`
-        <!---
-        TODO: Consider instead making tag-specific commits that check-in the artifacts. For now, the 
-        seamless AAR reproducibility makes this a non-issue.
-        -->
-        * `--allow-dirty` is required because we don't check-in the generated Maven local repository at this time.
+        * `--allow-dirty` is required because we don't check-in the generated Maven local repository.
     6. (Optional) Inspect extracted archive to ensure the local Maven repository artifacts are present
         1. Un-tar the `rustls-platform-verifier-android-*.crate` file inside of `target/package`.
         2. Verify `maven/rustls/rustls-platform-verifier` contains a single `*.RELEASE` directory and that contains a `.aar` file.
         3. (Optional) If the releaser has an external Gradle project that uses the configuration from the README, paste the path to the
            unzipped package's `Cargo.toml` as a replacement for the `manifestPath` variable. Run a Gradle Sync and observe everything works.
-    7. Publish the Android artifacts' new version: `cargo publish -p rustls-platform-verifier-android --alow-dirty`
+    7. **Ensure that all version changes are commited to the correct branch before proceeding**. All version increases should be checked-in prior
+       to publishing on crates.io.
+    8. Publish the Android artifacts' new version: `cargo publish -p rustls-platform-verifier-android --alow-dirty`
+
 3. Commit main crate's version increase on the release branch
-4. Publish the main crate's new version: `cargo publish -p rustls-platform-verifier`
+4. **Ensure that all version changes are commited to the correct branch before proceeding**. All version increases should be checked-in prior
+    to publishing on crates.io.
+5. Publish the main crate's new version: `cargo publish -p rustls-platform-verifier`
     * Do **not** use `--allow-dirty` for the main crate. Only the Android component requires it and a dirty workspace elsewhere is an error.
-
-See the Rustls repo [RELEASING] guidance for more information (e.g. on best practices for creating a GitHub release with a changelog).
+6. Follow the remaining steps in [RELEASING] to create the appropiate version tag.
+7. If a new Android component release was made: Before creating the GitHub release, run `./ci/archive_android_release.sh` to create a reproducible archive
+   containing the Android Maven components that were just published to crates.io. After creating the archive, upload it as an additional release artifact on GitHub.
```

Comment:
> upload it as an additional release artifact on GitHub.

Is this just a matter of navigating to the GitHub release, editing it, and doing a file upload? I haven't done this before.

Reply:
You should be able to add the artifacts when creating the release with the file picker/drag area at the bottom:

<img width="1024" alt="image" src="https://github.com/rustls/rustls-platform-verifier/assets/68570223/676a2fcd-54a4-494a-94d0-543ca9704cbc">

That's what I did for 0.1, at least.

Reply:
Perfect - thanks!